pkg/alertmanager/alertstore/store_test.go

```diff
@@ -52,7 +53,7 @@ func TestAlertStore_SetAndGetAlertConfig(t *testing.T) {
 		// The user has no config.
 		{
 			_, err := store.GetAlertConfig(ctx, "user-1")
-			assert.Equal(t, alertspb.ErrNotFound, err)
+			assert.Equal(t, chunk.ErrStorageObjectNotFound, errors.Cause(err))
```

Comment:
This doesn't look right. This test should check for generic error, and not error from one specific implementation of alertstore.

Reply:
Reverted the not-found case to be a generic error. Now the verbose errors are only used in failure cases like the file failed to be parsed.